### ISSUE TITLE
Nested Sprite crashes Windows target.

### ISSUE THREAD

**Issue by jgranick** [openfl/openfl-native#192]
#192 Issue by __jagt__,

Hi, I've reproduce the issue with a smaller test case.

```
package;


import flash.display.Bitmap;
import flash.display.BitmapData;
import flash.display.Sprite;
import openfl.Assets;
import openfl.display.FPS;


class Outer extends Sprite {
    public static var RADIUS : Float = 30.0;
    public static var LINE_WIDTH : Float = 2.0;
    public static var HIT_RADIUS : Float = 40.0;

    public function new() {
        super();
        var inner1 = new Sprite();
        var inner2 = new Sprite();

        inner1.graphics.lineStyle(LINE_WIDTH, 0x000000);
        inner1.graphics.drawCircle(0, 0, RADIUS);
        inner1.graphics.moveTo(-RADIUS, 0);
        inner1.graphics.lineTo(RADIUS, 0);

        inner2.graphics.beginFill(0xDDDDDD, 0.8);
        inner2.graphics.drawCircle(0, 0, HIT_RADIUS);

        addChild(inner1);
        addChild(inner2);
    }
}

class Main extends Sprite {
    public function new () {
        super ();
        var outer = new Outer();
        addChild( outer );
        addChild (new FPS());
    }


}
```

This would crash Windows target. It won't always happen but on my machine the chance is higher than 1/2. Notice that it doesn't need to be `FPS` class, seems `TextField` would have the same effects.

Here's my `haxelib list`:

```
lime: 0.9.2 0.9.3 [0.9.4]
lime-tools: 1.2.6 1.2.7 1.2.8 [1.2.9]
openfl: [1.2.2]
openfl-bitfive: [1.0.5]
openfl-compatibility: [1.0.1]
openfl-html5: 1.0.3 1.0.4 [1.0.5]
openfl-html5-dom: 1.1.0 1.1.1 [1.2.0]
openfl-native: 1.1.4 1.2.0 [1.2.2]
openfl-samples: 1.0.0 1.0.0-rc.3 1.1.0 1.2.0 [1.2.1]
openfl-tools: [1.0.10]
```

Thanks a lot for your works.

**Comment by jgranick** [comment]
Comment by __jagt__:

Just found a similar issue dated back to NME: http://www.openfl.org/archive/community/bugs/fps-crashes-windows-release/

I'd like to help track this down. Any idea about where should I look at?

**Comment by jgranick** [comment]
Comment by __Gama11__:

You could try isolating the crash step by step by removing things from [FPS.hx](https://github.com/openfl/openfl/blob/master/openfl/display/FPS.hx).

**Comment by jgranick** [comment]
Comment by __jagt__:

I just found that it happens whenever I add a TextField into the stage...

**Comment by jgranick** [comment]
Comment by __jagt__:

Then I found it's basically that I can't add anything to stage... This should be project specific, closing this  for now. Sorry for the trouble.

**Comment by jgranick** [comment]
Comment by __jagt__:

Updated issue body with a testcase and reopen.

**Comment by jgranick** [comment]
Comment by __jagt__:

Somehow this one is fixed in the 1.2.4 release. Thanks a lot.
